qa/tasks/mgr/dashboard/test_rgw.py

```diff
@@ -510,6 +510,7 @@ def test_get(self):
         self.assertStatus(200)
         self._assert_user_data(data)
         self.assertEqual(data['user_id'], 'admin')
+        self.assertTrue(data['stats'])
```

Comment:
Could we perform some validation on the stats so we ensure that the data displayed in the front-end is already there (luckily when the runtime type checker arrives, we'll have this validation performed within the dashboard, instead of externally)?